docker/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -ex
+
+gcloud config set project istio-test
```

Comment:
we should rename the file  gcloud_build.sh.
It is important to have a script that just works in our environment.